### ISSUE TITLE
Star Fox 64 3D visual artifacts

### ISSUE THREAD

**Issue by TheRealWario** [Lime3DS/Lime-3DS-Emulator#52]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Lime 2106

### Description of Issue

In Star Fox 64 3D, some particular levels and effects aren't rendered properly. From testing, I know it causes the levels Aquas, Solar, and 2nd boss phase of Venom 2 to render upside-down. Looking at it further, I noticed the reflections on the water of Corneria are also rendered upside down, and messes with the proper reflection effect. 
![Star Fox 64 3D_05 04 24_17 47 32 923](https://github.com/Lime3DS/Lime3DS/assets/119357717/84b758d6-e23c-4ead-9d04-583e624c1fd0)
![Star Fox 64 3D_05 04 24_17 48 04 66](https://github.com/Lime3DS/Lime3DS/assets/119357717/716ebc7c-c960-434f-8a36-483d1c0446fc)
![Star Fox 64 3D_05 04 24_17 50 07 736](https://github.com/Lime3DS/Lime3DS/assets/119357717/64c5e2cc-21ac-4a15-85e3-397a74dc93ee)

As with the change in leadership, i hope this issue can finally get the attention it needs.
Oh, and they already analyzed this issue in the lost Citra repository: https://web.archive.org/web/20201217075118/https://github.com/citra-emu/citra/issues/3104#issuecomment-591663930

### Expected Behavior

Well, i think having the level be rendered upside down is a bit strange, don't you think?

### Reproduction Steps

Go into the game with any graphics driver, play on Aquas and/or Solar, and witness

### Log File

[citra_log.txt](https://github.com/Lime3DS/Lime3DS/files/14891256/citra_log.txt)

### System Configuration

CPU: AMD Ryzen 5 5600H
GPU/Driver: NVIDIA GeForce GTX 1650, Vulkan, OpenGL, Software
RAM: 24gb
OS: Windows 11

**Comment by Gamer64ytb** [comment]
Does it happen for vulkan as well? If OpenGL only, I may know a workaround.

**Comment by TheRealWario** [comment]
No, this happens with every backend available. The bug is tied to the game itself, and is caused by how it creates an internal frame buffer upside down, and the game would flip it manually when displaying it. Here, it’s failing to flip its axis and results in the game displaying it incorrectly.

**Comment by Silvia-Dragoness** [comment]
I seem to recall the same thing happening with the original Star Fox 64 in the Project64 emulator, with any level that had funky distortion filters--Solar, Aquas, and the Warp Zones come to mind.

**Comment by Zombiegenocidest** [comment]
Not sure if anyone's working on this yet, but I can copy over all of the expected behaviour and in-depth logs/reports from my old issues thread from citra if it would assist.

**Comment by Rickroll10000** [comment]
is there any hope of this ever being fixed?

**Comment by DanGLVK** [comment]
The graphical artifacts are still very much present when it comes to water reflections even on the latest Alpha

**Comment by TheRealWario** [comment]
> The graphical artifacts are still very much present when it comes to water reflections even on the latest Alpha

Can you send a screenshot?

**Comment by DanGLVK** [comment]
> > The graphical artifacts are still very much present when it comes to water reflections even on the latest Alpha
> 
> Can you send a screenshot?

![Image](https://github.com/user-attachments/assets/42a7da27-0f27-41e1-8493-1e47d77f603f)

Water reflections are only displayed right in front of the player (with a clear separation) 

![Image](https://github.com/user-attachments/assets/7b3c5aef-73f4-430f-9979-cf6388f83a38)

This is how it's supposed to look like

This issue occurs regardless of renderer (OpenGL/Vulkan) or accurate multiplication (on/off) 

This issue can also be observed on the first stage, so replicating it should be easy

**Comment by UnownJ** [comment]
It would be great if someone could find a fix for this. I would be willing to send some money to help encourage it.
